src/components/Container/Container.tsx

```diff
@@ -2,7 +2,7 @@ import styled from "styled-components";
 import { HTMLAttributes } from "react";
 import { Orientation } from "@/components";
 
-type AlignItemsOptions = "start" | "center" | "end";
+type AlignItemsOptions = "start" | "center" | "end" | "stretch";
```

Comment:
Is it for align-items: stretch or align-self: stretch?

Reply:
We don't have `align-self` set up for this component yet.